crates/eframe/src/native/run.rs

```diff
@@ -189,13 +189,8 @@ fn run_and_return(event_loop: &mut EventLoop<UserEvent>, mut winit_app: impl Win
 
             if let Some(window) = winit_app.window(*window_id) {
                 log::trace!("request_redraw for {window_id:?}");
-                let is_minimized = window.is_minimized().unwrap_or(false);
-                if is_minimized {
-                    false
-                } else {
-                    window.request_redraw();
-                    true
-                }
+                window.request_redraw();
+                true
```

Comment:
This reverts the changes you added in https://github.com/emilk/egui/pull/3985

Reply:
ne. gyesog `request_repaint()` leul hochulhaji anhneundamyeon gwaenchanhseubnida.
44 / 5,000
Yes, it's ok as long as you don't keep calling `request_repaint()`.

Reply:
If someone want to keep calling `request_repaint()` but not call it when `minimized`, someone can do the following.
It works fine.

```
            let is_maximized = ui.input(|i| i.viewport().maximized.unwrap_or(false));
            if !is_minimized {
                ui.ctx().request_repaint();
            }
```